tests/dummy/config/environment.js

```diff
@@ -3,19 +3,15 @@
 module.exports = function (environment) {
   let ENV = {
     modulePrefix: 'dummy',
-    podModulePrefix: 'dummy/pods',
```

Comment:
When I reintroduce the documentations, I will be assuming the default Octane folder structure.